deps/wazuh_testing/wazuh_testing/qa_docs/lib/code_parser.py

```diff
@@ -70,13 +73,10 @@ def parse_comment(self, function):
 
         except Exception as inst:
             if hasattr(function, 'name'):
-                warnings.warn(f"Failed to parse comment of function '{function.name}'' from module {self.scan_file}. \
-                              Error: {inst}", stacklevel=2)
-                logging.warning(f"Failed to parse comment of function '{function.name}'' from module {self.scan_file}. \
-                                Error: {inst}")
+                CodeParser.LOGGER.warning(f"Failed to parse comment of function {function.name} "
```

Comment:
As the comment that is parsed is the documentation, how about this:
```suggestion
                CodeParser.LOGGER.warning(f"Failed to parse test documentation in {function.name} "
```

Reply:
Done in: https://github.com/wazuh/wazuh-qa/commit/86e5045beb49f8e91dbb42b6e59f8d3164a57523